packages/fxa-auth-server/fxa-oauth-server/lib/db/mysql/index.js

```diff
@@ -606,12 +614,22 @@ MysqlStore.prototype = {
     clientId,
     uid
   ) {
-    const res = await this._write(DELETE_REFRESH_TOKEN_WITH_CLIENT_AND_UID, [
-      buf(refreshTokenId),
-      buf(clientId),
-      buf(uid),
+    const deleteAccessTokens = this._write(
+      DELETE_ACTIVE_TOKENS_BY_CLIENT_AND_UID,
+      [buf(clientId), buf(uid)]
+    );
+
+    const deleteRefreshTokens = this._write(
+      DELETE_REFRESH_TOKEN_WITH_CLIENT_AND_UID,
+      [buf(refreshTokenId), buf(clientId), buf(uid)]
+    );
+
+    const [, refreshTokenRes] = await P.all([
```

Comment:
Should this be done in parallel? Is there a scenario where deleting a refresh tokens could fail and we don't want to delete access tokens?

Since this is parallel it is more performant though.

Reply:
Good suggestion. This is a pretty low traffic route, so doing those serially should be fine.